src/HandleErrors.js

```diff
@@ -0,0 +1,23 @@
+function handleErrors(textToConvert, toCase) {
+  const errors = [];
+  const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  if (!textToConvert) {
+    errors.push({ message: 'Text to convert is required.'
+    + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
+  }
+
+  if (!toCase) {
+    errors.push({ message: '"toCase" query param is required.'
+    + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
+  } else {
+    if (!supportedCases.includes(toCase)) {
+      errors.push({ message: 'This case is not supported.'
+      + ' Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.' });
+    }
+  }
+
+  return errors;
```

Comment:
you can create separate file with object: errorMessages, and take only object's keys here : 
`error.push({ message: errorMessage.toCaseEmpty })`;